### ISSUE TITLE
no module named caffe

### ISSUE THREAD

**Issue by prmex900** [phillipi/pix2pix#156]
running step 4 python batch_hed.py --images_dir=/data/photos/ --hed_mat_dir=/data/photos1

but always get "Import Error: no module named caffe"

what do you suggest?

**Comment by junyanz** [comment]
You need to install the [caffe](http://caffe.berkeleyvision.org/installation.html) library first. Also, see the original HED [repo](https://github.com/s9xie/hed).

**Comment by Praddmobile** [comment]
> You need to install the [caffe](http://caffe.berkeleyvision.org/installation.html) library first. Also, see the original HED [repo](https://github.com/s9xie/hed).

i actually did it. i followed all the instructions from original repo. 
module is still missing. maybe I had to link it somehow?
i tried all the steps with and without anaconda. I also made clear homebrew intall and clear opencv install. i deleted everything from usr/local before installing these.
using os Mojave

**Comment by junyanz** [comment]
Have you added your pycaffe path to your environment?

**Comment by Praddmobile** [comment]
> Have you added your pycaffe path to your environment?

i think no..
how to do it?

**Comment by Praddmobile** [comment]
> See these two posts [1](http://caffe.berkeleyvision.org/tutorial/interfaces.html) and [2](https://github.com/BVLC/caffe/issues/2819)

yes I was about to do that. but problem is I don't know which path to use
I have 3 caffe folders from original s9xie/hed repo.
i linked it to hed-master/python/caffe

Am I going right way?

**Comment by junyanz** [comment]
`PYTHONPATH=/path/to/caffe/python:`

**Comment by Praddmobile** [comment]
> `PYTHONPATH=/path/to/caffe/python:`

can you check your .edu email? I dropped you full description on this issue
